docs/excel/custom-functions-authentication.md

```diff
@@ -10,9 +10,9 @@ In some scenarios your custom function will need to authenticate the user in ord
   
 ## AsyncStorage object
 
-The custom functions runtime doesn't have a `localStorage` object available on the global window, where you might typically store data. Instead, you should share data between custom functions and task panes, by using [OfficeRuntime.AsyncStorage](https://docs.microsoft.com/javascript/api/office-runtime/officeruntime.asyncstorage) to set and get data. 
+The custom functions runtime doesn't have a `localStorage` object available on the global window, where you might typically store data. Instead, you should share data between custom functions and task panes, by using [OfficeRuntime.AsyncStorage](https://docs.microsoft.com/javascript/api/office-runtime/officeruntime.asyncstorage) to set and get data.
```

Comment:
Just flagging that we won't know if "runtime" is the correct word here until after the meeting today. :)

Reply:
Renamed to "custom function runtime" throughout which was decision in the meeting.